pinot-distribution/pom.xml

```diff
@@ -262,6 +262,43 @@
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-enforcer-plugin</artifactId>
           </plugin>
+          <!-- Create symlink "build" to the root directory (Unix only) -->
+          <plugin>
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>1.5.0</version>
+            <executions>
+              <execution>
+                <id>remove-build-directory</id>
+                <phase>clean</phase>
```

Comment:
Will it fail if user doesn't do `mvn clean package`, but `mvn package`? If not, then we don't need this?

Reply:
i haven't tested this but removing symlinks before recreating it will do no harm anyways. let's keep this